library/common/config_template.cc

```diff
@@ -233,33 +209,10 @@ stats_flush_interval: {{ stats_flush_interval_seconds }}s
             regex: '^http.hcm.decompressor.*'
         - safe_regex:
             google_re2: {}
-            regex: 'http.hcm.downstream_rq_[1|2|3|4|5]xx'
-        - exact: 'http.hcm.downstream_rq_total'
-        - exact: 'http.hcm.downstream_rq_completed'
-        - safe_regex:
-            google_re2: {}
-            regex: 'vhost.api.vcluster\.[\w]+?\.upstream_rq_[1|2|3|4|5]xx'
-        - safe_regex:
-            google_re2: {}
-            regex: 'vhost.api.vcluster\.[\w]+?\.upstream_rq_retry'
-        - safe_regex:
-            google_re2: {}
-            regex: 'vhost.api.vcluster\.[\w]+?\.upstream_rq_retry_limit_exceeded'
-        - safe_regex:
-            google_re2: {}
-            regex: 'vhost.api.vcluster\.[\w]+?\.upstream_rq_retry_overflow'
-        - safe_regex:
-            google_re2: {}
-            regex: 'vhost.api.vcluster\.[\w]+?\.upstream_rq_retry_success'
-        - safe_regex:
-            google_re2: {}
-            regex: 'vhost.api.vcluster\.[\w]+?\.upstream_rq_time'
-        - safe_regex:
-            google_re2: {}
-            regex: 'vhost.api.vcluster\.[\w]+?\.upstream_rq_timeout'
+            regex: '^http.hcm.downstream_rq_(?:[12345]xx|total|completed)'
```

Comment:
should total and completed be kept as exact?

Reply:
It'll still be cheaper to check the shared prefix once via the single regex.